packages/graphql/tests/e2e/subscriptions/auth/where.int.test.ts

```diff
@@ -0,0 +1,217 @@
+    });
+
+    test("only return events matching the static where statement", async () => {
+        wsClient = new WebSocketTestClient(server.wsPath, jwtToken);
+        await wsClient.subscribe(`
+            subscription {
+                ${typeUser.operations.subscribe.created} {
+                    ${typeUser.operations.subscribe.payload.created} {
+                        name
+                    }
+                }
+            }
+            `);
+
+        await createUser("user1");
+        await createUser("arthur");
+
+        expect(wsClient.events).toEqual([
+            {
+                [typeUser.operations.subscribe.created]: {
+                    [typeUser.operations.subscribe.payload.created]: {
+                        name: "arthur",
+                    },
+                },
+            },
+        ]);
+        expect(wsClient.errors).toEqual([]);
+    });
+
+    test("only return events matching the jwt where statement", async () => {
+        wsClient = new WebSocketTestClient(server.wsPath, jwtToken);
+        await wsClient.subscribe(`
+            subscription {
+                ${typeMovie.operations.subscribe.created} {
+                    ${typeMovie.operations.subscribe.payload.created} {
+                        title
+                    }
+                }
+            }
+            `);
+
+        await createMovie("movie1");
+        await createMovie("movie2");
+
+        expect(wsClient.events).toEqual([
+            {
+                [typeMovie.operations.subscribe.created]: {
+                    [typeMovie.operations.subscribe.payload.created]: {
+                        title: "movie2",
+                    },
+                },
+            },
+        ]);
+        expect(wsClient.errors).toEqual([]);
+    });
+
+    test("only return events matching the auth where statement and where argument", async () => {
+        wsClient = new WebSocketTestClient(server.wsPath, jwtToken);
+        await wsClient.subscribe(`
+            subscription {
+                ${typeUser.operations.subscribe.created}(where: {surname: "Conan Doyle"}) {
+                    ${typeUser.operations.subscribe.payload.created} {
+                        name
+                        surname
+                    }
+                }
+            }
+            `);
+
+        await createUser("arthur", "Dent");
+        await createUser("arthur", "Conan Doyle");
+        await createUser("Marvin", "Conan Doyle");
+
+        expect(wsClient.events).toEqual([
+            {
+                [typeUser.operations.subscribe.created]: {
+                    [typeUser.operations.subscribe.payload.created]: {
+                        name: "arthur",
+                        surname: "Conan Doyle",
+                    },
+                },
+            },
+        ]);
+        expect(wsClient.errors).toEqual([]);
+    });
+
+    test("where option in auth takes preference over where arg", async () => {
```

Comment:
I assume this should be `precedence``
```suggestion
    test("where option in auth takes precedence over where arg", async () => {
```